### ISSUE TITLE
`pan` event (renamed from `slide`)

### ISSUE THREAD

**Issue by enjibby** [pisi/Reel#51]
When both jquery, reel and mootools are loaded onto the same page and the user attempts to slide the reel to the left or right with the mouse, this accidently triggers the mootools 'slide' event on the outer div, which causes the image to disappear like a raised curtain. If possible, could the 'slide' event be renamed to something a little less generic to counter this issue? Thanks!

**Comment by pisi** [comment]
Oh, sorry to hear that. Didn't know about this conflict until now, thanks! I will sure rename the event. Once again I would say - it originaly was `drag`, but that was found to conflict in IE ;)

Now just find a proper name. It should best refer to some two-dimensional task (it carries along `x` and `y` arguments). It is reused twice in Reel (by mouse mouse and touch drag). Any ideas?

**Comment by enjibby** [comment]
Actually, in the copy I used on my site, I just renamed it to 'reelslide', but if it's a two-dimensional task my guess would be that it's something like a 'pan'. Also, given that this is primarily to show images from different angles, might it also be correct to use 'rotate'? Granted both of these words are again quite generic, perhaps namespacing them in some way such as 'reel.slide', 'reel-pan' or 'reel_rotate'? These are just suggestions, as I don't know too much about the standards involved with jQuery (not mentioning javascript in general) so implementation is well and truly up to you :)
